src/aat/resources/features/F-044/F-044.feature

```diff
@@ -0,0 +1,136 @@
+@F-044
+Feature: F-044: Submit event creation as Case worker
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-278
+  Scenario: must submit the event creation successfully for correct inputs
+    Given a user with [an active profile in CCD]
+    And a case that has just been created as in [Standard_Full_Case_Creation_Data]
+    And a successful call [to get an event token for the case just created] as in [F-044-Prerequisite]
+    When a request is prepared with appropriate values
+    And the request [contains a case Id that has just been created as in Standard_Full_Case_Creation_Data]
+    And the request [contains an event token for the case just created above]
+    And it is submitted to call the [submit event creation as case worker] operation of [CCD Data Store]
+    Then a positive response is received
+    And the response [contains the updated case details, along with an HTTP 201 Created]
+    And the response has all other details as expected
+
+  @S-279
+  Scenario: must return negative response when request does not provide valid authentication credentials
+    Given a user with [an active profile in CCD]
+    When a request is prepared with appropriate values
+    And the request [does not provide valid authentication credentials]
+    And it is submitted to call the [submit event creation as case worker] operation of [CCD Data Store]
+    Then a negative response is received
+    And the response [contains an HTTP 403 Forbidden]
```

Comment:
above scenario should retrun 401 
@Ignore # Response code mismatch, expected: 401, actual: 403